### DISCUSSION TITLE
CPU usage on Ubuntu vs Windows

### DISCUSSION THREAD

**Discussion by domis86** [pixijs/pixijs#7903] [Q&A]
Hi
I would like to understand why on my Ubuntu 20 there is high CPU usage on any page with canvas (animation).
This is not strictly pixi.js related problem.

Example: https://pixijs.io/examples/#/demos-basic/container.js

On Windows: 1 to 2 % CPU usage - thats good.
On Ubuntu 20.04.2:
(my laptop is: 32 GB RAM, CPU AMD Ryzen 7 4800h, Geforce RTX 2060)

![ubuntu-cpu-gpu-1](https://user-images.githubusercontent.com/2327600/138554948-142240e3-6fb9-4c18-81f6-8d1115cdf063.png)

^ note: this is Chrome, but with Firefox is similar.
^ note: please notice that GPU is also used.

Also happens with canvas without pixi.js - example: https://codepen.io/ksenia-k/full/gOPboQg

Questions:
* does anyone have similar CPU usage on Ubuntu when canvas is used on page in browser?
    * if you will check this then please keep browser window with canvas visible and focused - if is hidden/minimized then cpu usage is low.
* why there is such difference between Ubuntu and Windows?

**Comment by bigtimebuddy** [comment]
Sometimes for Linux, outdated or generic GPU drivers can be a source of problems. Maybe make sure your video card has up-to-date drivers?

**Reply by domis86** [reply]
so on your computer you dont have this problem?

**Reply by bigtimebuddy** [reply]
I don't use either of these platforms regularly. Mostly am reflecting some general advice I've heard on GitHub issues.
